css/mediaqueries/test_media_queries.html

```diff
@@ -607,11 +607,11 @@
     }, "Sanity check for overflow-inline");
 
     // Parsing tests for update from mediaqueries-4
-    expression_should_be_parseable("update")
-    expression_should_be_parseable("update: none")
-    expression_should_be_parseable("update: slow")
-    expression_should_be_parseable("update: fast")
-    expression_should_not_be_parseable("update: some-random-invalid-thing")
+    expression_should_be_known("update")
+    expression_should_be_known("update: none")
+    expression_should_be_known("update: slow")
+    expression_should_be_known("update: fast")
+    expression_should_not_be_known("update: some-random-invalid-thing")
```

Comment:
This should be `expression_should_be_unknown` rather than `expression_should_not_be_known`.  Otherwise LGTM.

Reply:
Thanks! updated